src/zkevm_specs/state.py

```diff
@@ -222,25 +225,25 @@ def check_stack(row: Row, row_prev: Row):
 def check_storage(row: Row, row_prev: Row, tables: Tables):
     get_addr = lambda row: row.keys[2]
     get_storage_key = lambda row: row.keys[4]
-
-    # TODO: cold VS warm
-    # TODO: connection to MPT on first and last access for each (address, key)
+    get_committed_value = lambda row: row.auxs[0]
 
     # 0. Unused keys are 0
-    assert row.keys[1] == 0
     assert row.keys[3] == 0
 
-    # 1. First access for a set of all keys
-    #
-    # We add an extra write to set the value of the state in previous block, with rwc=0.
-    #
-    # When the set of all keys changes (first access of storage (address, key))
-    # - It must be a WRITE
-    if not all_keys_eq(row, row_prev):
-        assert row.is_write == 1 and row.rw_counter == 0
+    # 1. When keys don't change, committed_value must be kept equal
+    if all_keys_eq(row, row_prev):
+        assert get_committed_value(row) == get_committed_value(row_prev)
```

Comment:
Since now `tx_id` is also part of keys, when `tx_id` changes, we need to `assert get_committed_value(row) == row_prev.value` to make sure the `committed_value` is updated.